src/DataAccess/DbalCommentRepository.php

```diff
@@ -33,9 +33,20 @@ function( Spenden $spenden ) {
 					->setAuthorName( $spenden->getName() )
 					->setCommentText( $spenden->getKommentar() )
 					->setDonationAmount( (float)$spenden->getBetrag() )
-					->setPostingTime( $spenden->getDtNew() );
+					->setPostingTime( $spenden->getDtNew() )
+					->setDonationId( $spenden->getId() )
+					->freeze()
+					->assertNoNullFields();
 			},
-			$this->entityRepository->findBy( [ 'isPublic' => true ], null, $limit )
+			$this->getSpenden( $limit )
+		);
+	}
+
+	private function getSpenden( int $limit ): array {
```

Comment:
Should'nt that be called getDonations, even when the entity is called Spenden (until we use the 2.x branch)?